### PR TITLE
prev/next segment button features

### PR DESCRIPTION
**Pull request by sxian0613**
* If the current segment is the first segment of the day, disable the prev-segment button.
* If the current segment is the last segment of the day, disable the next-segment button.

### CODE REVIEW

File: packages/web-components/src/player-component/player.class.ts [hunk 1]
```diff
@@ -438,6 +438,46 @@ export class PlayerWrapper {
...
+            }
+        }
+    }
+
+    private updateNextSegmentButton() {
+        const lastSegmentStartSeconds = extractRealTimeFromISO(
+            this._availableSegments?.timeRanges[this._availableSegments?.timeRanges.length - 1]?.start,
+            this._currentDate
+        );
+        if (lastSegmentStartSeconds === this.currentSegment.startSeconds) {
```
Comment (giakas) [inline packages/web-components/src/player-component/player.class.ts:462]: this.disableNextSegmentButton(lastSegmentStartSeconds === this.currentSegment.startSeconds)

File: packages/web-components/src/player-component/player.class.ts [hunk 2]
```diff
@@ -438,6 +438,46 @@ export class PlayerWrapper {
...
+
+    private disablePrevSegmentButton(disable: boolean) {
+        for (const element of this.controls?.elements_) {
+            if (element?.isPrevSegmentButton) {
+                element.disablePrevSegmentButton(disable);
+            }
+        }
+    }
+
+    private updateNextSegmentButton() {
```
Comment (giakas) [inline packages/web-components/src/player-component/player.class.ts:457]: these are too small and not being used anywhere else and called one after another,, why not merge them to single function called updatePrevNextSegmentButtonsDisabledState ?

File: packages/web-components/src/player-component/player.class.ts (hunk 1 above)
Comment (sxian0613) [reply packages/web-components/src/player-component/player.class.ts]: Change is made.

File: packages/web-components/src/player-component/player.class.ts (hunk 2 above)
Comment (sxian0613) [reply packages/web-components/src/player-component/player.class.ts]: The two functions are combined to updatePrevNextSegmentButtonsDisabledState.

**Review by giakas — APPROVED** [review]

**Pull request merged**